app/controllers/sessions_controller.rb

```diff
@@ -3,7 +3,7 @@
 class SessionsController < ApplicationController
   layout 'auth', only: %i[new create]
 
-  before_action :redirect_to_home, if: :logged_in?, only: %i[new create]
+  before_action :redirect_to_home, if: :require_redirection?, only: %i[new create destroy]
```

Comment:
How about replace `require_redirection?` to `redirection_to_home?`

Reply:
Fixed in b0f5d7a98e9434e3a6ae272b9d2de662e8bbc2bd